articles/architecture-scenarios/_includes/_multitenancy.md

```diff
@@ -0,0 +1,3 @@
+Many B2B platforms implement some form of isolation and/or branding for their customers' organization.  This can add complexity to any IAM system.  If you this applies to you, you should take some time to read through our guidance around this type of environment.
```

Comment:
```suggestion
Many B2B platforms implement some form of isolation and/or branding for their customers' organization, and this can add complexity to any Identity and Access Management (IAM) system.  If you this applies to you, then we recommend you take some time to read through our guidance and best practice advice concerning this type of environment.
```

Reply:
```suggestion
Many B2B platforms implement some form of isolation and/or branding for their customers' organization, and this can add complexity to any Identity and Access Management (IAM) system.  If this applies to you, then we recommend you take some time to read through our guidance and best practice advice concerning this type of environment. 
```